consensus/types/src/attestation/payload_attestation.rs

```diff
@@ -0,0 +1,27 @@
+use crate::attestation::payload_attestation_data::PayloadAttestationData;
+use crate::test_utils::TestRandom;
+use crate::*;
```

Comment:
We're now preferring no wildcard imports in consensus type crates, see:
* https://github.com/sigp/lighthouse/pull/7827

Reply:
Fixed in 22060ca1